### ISSUE TITLE
Uncaught Error: Aligner: aligner-php has already been activated

### ISSUE THREAD

**Issue by cblaupanda** [adrianlee44/atom-aligner#35]
[Enter steps to reproduce below:]

1. ... install update
2. ...

**Atom Version**: 1.0.19
**System**: Ubuntu 14.04.3
**Thrown From**: [aligner](https://github.com/adrianlee44/atom-aligner) package, v0.17.0

### Stack Trace

Uncaught Error: Aligner: aligner-php has already been activated

```
At /home/christoph/.atom/packages/aligner/lib/provider-manager.coffee:5

Error: Aligner: aligner-php has already been activated
    at Error (native)
    at ProviderManager.register (/home/christoph/.atom/packages/aligner/lib/provider-manager.coffee:5:13)
    at Aligner.registerProviders (/home/christoph/.atom/packages/aligner/lib/main.coffee:49:77)
    at Provider.module.exports.Provider.provide (/opt/atom/resources/app.asar/node_modules/service-hub/lib/provider.js:30:52)
    at /opt/atom/resources/app.asar/node_modules/service-hub/lib/service-hub.js:55:26
    at doNTCallback0 (node.js:416:9)
    at process._tickCallback (node.js:345:13)
```

### Commands

```
     -9:22.1.0 aligner:align (atom-text-editor.editor.is-focused)
     -9:13.4.0 core:save (atom-text-editor.editor.is-focused)
     -7:34.1.0 find-and-replace:show (atom-text-editor.editor.is-focused)
     -7:31.3.0 core:confirm (atom-text-editor.editor.mini.is-focused)
     -7:30.3.0 editor:consolidate-selections (atom-text-editor.editor.mini.is-focused)
     -7:30.2.0 core:cancel (atom-text-editor.editor.mini.is-focused)
     -7:30 editor:consolidate-selections (atom-text-editor.editor.is-focused)
     -7:30 core:cancel (atom-text-editor.editor.is-focused)
     -7:29.9.0 editor:consolidate-selections (atom-text-editor.editor.is-focused)
     -7:29.9.0 core:cancel (atom-text-editor.editor.is-focused)
     -7:07.4.0 core:select-right (atom-text-editor.editor.is-focused)
     -0:14.4.0 settings-view:check-for-package-updates (atom-workspace.workspace.scrollbars-visible-always.theme-atom-monokai.theme-one-dark-ui)
```

### Config

```json
{
  "core": {
    "themes": [
      "one-dark-ui",
      "atom-monokai"
    ]
  }
}
```

### Installed Packages

```coffee
# User
aligner, v0.17.0
aligner-php, v1.0.0
aligner-puppet, v0.1.1
atom-monokai, v0.9.2
language-puppet, v0.14.0
save-session, v0.15.7

# Dev
No dev packages
```

**Comment by dennis-best** [comment]
I got the same error. I noticed that I had aligner-ruby installed and activated. So I just uninstalled it and reinstalled aligner, then reinstalled aligner-ruby (and other aligner packages) and all is good.

**Comment by adrianlee44** [comment]
Thanks for reporting the bug. Restarting Atom will fix this bug.

It was due to improper handling of activation and deactivation of aligner. This is patched in v0.17.0 and should be fixed from v0.17.0 onward.
